src/plane.js

```diff
@@ -2,11 +2,11 @@ class Plane {
     tailNumber;
 
     constructor(tailNumber) {
-        this.tailNumber = Math.random() * 100
+        this.tailNumber = Math.round(Math.random() * (1000 - 9999) + 9999);
```

Comment:
Realised that I wanted a 4 digit number, as mentioned before I found this to difficult to deal with later so removed it.